ui/pages/settings/transaction-shield-tab/submit-claim-form/submit-claim-form.tsx

```diff
@@ -458,30 +426,74 @@ const SubmitClaimForm = () => {
         width={BlockSize.Full}
       />
       <FormTextField
-        label={`${t('shieldClaimImpactedWalletAddress')}*`}
+        label={`${t('shieldClaimReimbursementWalletAddress')}*`}
         placeholder={'e.g. 0x71C...B5f6d'}
         inputProps={{
-          'data-testid': 'shield-claim-impacted-wallet-address-input',
+          'data-testid': 'shield-claim-reimbursement-wallet-address-input',
         }}
         helpTextProps={{
-          'data-testid': 'shield-claim-impacted-wallet-address-help-text',
+          'data-testid': 'shield-claim-reimbursement-wallet-address-help-text',
           color: DsTextColor.textAlternative,
         }}
         helpText={
-          errors.impactedWalletAddress
-            ? errors.impactedWalletAddress.msg
-            : undefined
+          errors.reimbursementWalletAddress
+            ? errors.reimbursementWalletAddress.msg
+            : t('shieldClaimReimbursementWalletAddressHelpText')
         }
-        id="impacted-wallet-address"
-        name="impacted-wallet-address"
+        id="reimbursement-wallet-address"
+        name="reimbursement-wallet-address"
         size={FormTextFieldSize.Lg}
-        onChange={(e) => setImpactedWalletAddress(e.target.value)}
-        onBlur={() => validateImpactedWalletAddress()}
-        value={impactedWalletAddress}
-        error={Boolean(errors.impactedWalletAddress)}
+        onChange={(e) => setReimbursementWalletAddress(e.target.value)}
+        onBlur={() => validateReimbursementWalletAddress()}
+        value={reimbursementWalletAddress}
+        error={Boolean(errors.reimbursementWalletAddress)}
         required
         width={BlockSize.Full}
       />
+      {/* Incident details */}
+      <Box className="mt-4">
+        <Text variant={TextVariant.HeadingSm}>
+          {t('shieldClaimIncidentDetails')}
+        </Text>
+        <Text
+          variant={TextVariant.BodySm}
+          color={TextColor.TextAlternative}
+          className="mb-2"
+        >
+          {t('shieldClaimIncidentDetailsDescription')}
+        </Text>
+
+        <Box
+          borderColor={BoxBorderColor.BorderMuted}
+          className="w-full h-[1px] border border-b-0"
+        ></Box>
+      </Box>
+      <AccountSelector
+        label={`${t('shieldClaimImpactedWalletAddress')}*`}
+        modalTitle={t('shieldClaimSelectAccount')}
+        impactedWalletAddress={impactedWalletAddress}
+        onAccountSelect={(address) => {
+          setImpactedWalletAddress(address);
+          // only validate equality if reimbursement wallet address is set and valid format
+          if (
+            reimbursementWalletAddress &&
+            (!errors.reimbursementWalletAddress ||
+              errors.reimbursementWalletAddress.msg ===
+                t('shieldClaimSameWalletAddressesError'))
```

Comment:
Assuming that error codes are implemented later

Reply:
i'll save either errorCode/message on the errorState's message key. if the value is not in the map then it's a custom message from backend